tests/unit/pages/lesson-details/LessonDetails.spec.jsx

```diff
@@ -65,13 +70,25 @@ describe('LessonDetails', () => {
     expect(description).toBeInTheDocument()
   })
 
-  it('should open content', async () => {
+  it('should open and close content', async () => {
     const title = await screen.findByText('lesson.attachments')
 
     fireEvent.click(title)
 
     const attachment = screen.getByText('file1.png')
     expect(title).toBeInTheDocument()
-    expect(attachment).toBeInTheDocument()
+    expect(attachment).toBeVisible()
+
+    fireEvent.click(title)
+
+    await waitFor(() => {
+      expect(attachment).not.toBeVisible()
+    })
+  })
+
+  it('should ...', async () => {
```

Comment:
Why description of test is `should ...`?

Reply:
Thanks for pointing this out. Forgot to apply changes after stashing.